examples/gltf.rs

```diff
@@ -80,7 +83,7 @@ fn main() {
         }
         let yrot = cgmath::Quaternion::<f32>::from_angle_y(cgmath::Rad(-st.yaw));
         let xrot = cgmath::Quaternion::<f32>::from_angle_x(cgmath::Rad(-st.pitch));
-        cam.set_transform(cgmath::Point3::from_vec(st.position), yrot * xrot, 1.0);
```

Comment:
Time to write first person controls.

Reply:
There is one problem... I'll explain in gitter